src/appengine/handlers/testcase_detail/show.py

```diff
@@ -368,8 +368,14 @@ def get_testcase_detail(testcase):
   crash_state = testcase.crash_state
   crash_state_lines = crash_state.strip().splitlines()
   crash_type = data_handler.get_crash_type_string(testcase)
-  reproduction_help_url = data_handler.get_reproduction_help_url(
-      testcase, config)
+  formatted_reproduction_help = data_handler.get_formatted_reproduction_help(
+      testcase)
+  # TODO(mbarbella): Remove this once everything has been moved to HELP_FORMAT.
```

Comment:
can just remove reproduction_help_url completely too and add the field on staging internal, external, also update job types to have both HELP_URL and REPRO_HELP_TEMPLATE

Reply:
I don't think the case of having both makes too much sense. +1 to removing reproduction_help_url though. I think I'll just replace it all at once in this change.

Reply:
Based on our offline discussion, keeping both. Updated the comment to clarify that we prioritize the template.